docker/build.sh

```diff
@@ -4,27 +4,34 @@ set -ex
 
 DATE=$(date +%m.%y)
 
-docker build -t electronuserland/builder:base -t "electronuserland/builder:base-$DATE" docker/base
+docker build -t devalexandria/builder:base -t "devalexandria/builder:base-$DATE" docker/base
```

Comment:
I'm fully in favor of adding a node 20 image, but this needs to remain `electronuserland/____` in order for these images to be pushed to dockerhub correctly

Reply:
Sorry for this, I needed this for my personnal use and forgot to remove ref to my private repo.